### PR TITLE
Feature/component textinput storybook

### PR DESCRIPTION
**Pull request by Kischa-hub**
Adding the following Atoms with Storybook and making a Jest test
1- Input text 
2- Avatar

Adding the following Molocules with Storybook and making a Jest test
1- Uploading Image
2- User Info

### CODE REVIEW

**Comment by EmanFouad** [comment]
Great Work Kareem,
Please consider the following changes:

**for design :**
For all InputText:
  use the same style that exist in figma for :
* normal status , use the same border color that exist in default status in figma
* use the same placeholder at focus mode 
* use the same error icon

For InputText+LeftIcon:
* use the same size for icon and same space between icon and text

For InputImage:
* use the same size for background of letter

**for writing story:**
* you can add some functionality at inputs at onChange and add close icon that remove text when click on it

**Notes:**
* I reviewed design and writing stroy only , not testing (testing files doesn't appear to me ) 
* It's prefer to make testing in separate pr

**Review by EmanFouad — CHANGES_REQUESTED** [review]
Great Work Kareem,
Please consider the following changes:

**for design :**
For all InputText:
  use the same style that exist in figma for :
* normal status , use the same border color that exist in default status in figma
* use the same placeholder at focus mode 
* use the same error icon

For InputText+LeftIcon:
* use the same size for icon and same space between icon and text

For InputImage:
* use the same size for background of letter

**for writing story:**
* you can add some functionality at inputs at onChange and add close icon that remove text when click on it

**Notes:**
* It's prefer to make testing in separate pr

**Comment by Kischa-hub** [comment]
Done All Changes

File: storybook/src/kareem/Components/Atoms/Inputtexts/InputImage.test.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import InputImage from "./InputImage";
+//fullRound, width, icon, placeholder
+describe("Render the InputImage component", () => {
```
Comment (EmanFouad) [inline storybook/src/kareem/Components/Atoms/Inputtexts/InputImage.test.js:5]: Test Cases failed at this component , please check it again

**Review by EmanFouad — CHANGES_REQUESTED** [review]

File: storybook/src/kareem/Components/Atoms/Inputtexts/InputImage.test.js (hunk 1 above)
Comment (Kischa-hub) [reply storybook/src/kareem/Components/Atoms/Inputtexts/InputImage.test.js:5]: Please take the last updated Branch, I have already updated  all my snapshot .

Comment (Kischa-hub) [reply storybook/src/kareem/Components/Atoms/Inputtexts/InputImage.test.js:5]: I have updated all the snapshot files and upload all of them

**Review by EmanFouad — APPROVED** [review]
LGTM

**Pull request merged**